package.json

```diff
@@ -14,7 +14,7 @@
   "scripts": {
     "test": "jest --coverage --coverageDirectory ../coverage",
     "build": "rimraf dist && babel src --out-dir dist --extensions .ts --ignore src/**/*.test.ts && tsc --project tsconfig.build.json",
-    "lint:eslint": "eslint --ext .ts utils src",
+    "lint:eslint": "eslint --ext .ts utils src --max-warnings 0",
```

Comment:
Hardcore. I like it! :)

Reply:
Haha. This basically just treats warnings as errors. Ensures we don't miss anything